internal/alias/config.go

```diff
@@ -0,0 +1,129 @@
+package alias
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	scwAliasConfigPath = "SCW_ALIAS_CONFIG_PATH"
```

Comment:
Please add this to the config.md documentation

Reply:
CLI config use currently only used by aliases, `config` namespace is about the scw config